operations/file_operations.rb

```diff
@@ -0,0 +1,28 @@
+def save_items_to_json(items)
+  serialized_items = items.map(&:to_json)
+  write_json_to_file('items.json', serialized_items)
+end
+
+def write_json_to_file(file_name, data)
+  File.open(file_name, 'w') do |file|
+    file.puts data.to_json
+  end
+end
+
+def load_items_from_json
+  return [] unless File.exist?('items.json')
+
+  class_mapping = {
+    'Book' => Book,
+    'Label' => Label,
+    'MusicAlbum' => MusicAlbum,
+    'Genre' => Genre,
+    'Game' => Game,
+    'Author' => Author
+  }
+  json_data = JSON.parse(File.read('items.json'))
+  json_data.map do |item_data|
+    class_name = item_data['class_name']
+    class_mapping[class_name].from_json(item_data)
```

Comment:
- [ ] I also noticed that once `Quit` the console app and I tried to restart the app to check if Data was preserved and to also check whether your method to get back all the data that was created indeed works as expected, the app didn't start at all. So kindly fix that as well :+1: I have attached the error message below.
- [ ] So kindly ensure that Data is well Preserved in your console app and stored data from the `.json files` is loaded when the app starts. Kindly ensure that your app also starts without any errors :wink: 
- Error Message
![image](https://github.com/brighton0102/Catalog-of-my-things/assets/102699958/ba97b9a2-7222-4442-a776-24b2a4b5a4a8)